### ISSUE TITLE
Pakage management reports 'not installed' to installed package 

### ISSUE THREAD

**Issue by shevelev** [modxbot/migrate#4955]
shevelev created Redmine issue ID 4955

Managers page 'package management' shows wrong package installed status in the list.

**Comment by wshawn** [comment]
wshawn submitted:

I already reported this.

Reinstall in the Package Manager over the top of the other one, to straighten it out.

**Comment by shevelev** [comment]
shevelev submitted:

Well, I've done the reinstallation of some packages. Unfortunately there is no effect. No one has changed its status.

W ShawnWilkerson wrote:

> I already reported this.
> 
> Reinstall in the Package Manager over the top of the other one, to straighten it out.

**Comment by leoandrews** [comment]
leoandrews submitted:

I've also found problems with the package management list being out of sync with what's really going on. I downloaded Wayfinder using the Download Extras function in PM, I verified it was saved to disk but it never appeared in the package list ready for install.

Until... I cleared the whole of the site's cache - then it appeared. I suspect the problem we're seeing here, therefore, is caching related.
